packages/libs/react-ui/src/components/NavAccordion/NavAccordionGroup.tsx

```diff
@@ -0,0 +1,82 @@
+'use client';
+
+import {
+  navAccordionGroupButtonClass,
+  navAccordionGroupIconClass,
+  navAccordionGroupListClass,
+  navAccordionGroupListItemClass,
+  navAccordionGroupTitleClass,
+  navAccordionListClass,
+} from './NavAccordion.css';
+import type { INavAccordionLinkProps } from './NavAccordionLink';
+
+import {
+  accordionButtonClass,
+  accordionCollapse,
+  accordionExpand,
+} from '@components/Accordion/Accordion.css';
+import { SystemIcon } from '@components/Icon';
+import classNames from 'classnames';
+import type { FC, FunctionComponentElement } from 'react';
+import React, { Children, useState } from 'react';
+
+export interface INavAccordionGroupProps {
+  children: FunctionComponentElement<INavAccordionLinkProps>[];
+  index?: number;
+  onClose?: () => void;
+  onOpen?: () => void;
+  onClick?: () => void;
+  title: string;
+}
+
+export const NavAccordionGroup: FC<INavAccordionGroupProps> = ({
+  children,
+  onClose,
+  onOpen,
+  onClick,
+  title,
+}) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const handleClick = (): void => {
+    if (isOpen) {
+      onClose?.();
+    } else {
+      onOpen?.();
+    }
+    setIsOpen(!isOpen);
+    onClick?.();
```

Comment:
I forget. Why do we need a separate onClick handler in addition to the onOpen and onClose?

Reply:
We don't. It's more of an extra to offer more flexibility. What do you think, shall I remove it?